mmv1/products/privateca/api.yaml

```diff
@@ -419,7 +562,7 @@ objects:
         name: 'certificateDescription'
```

Comment:
I think subfields need to be `output: true` too. Terraform will do the right thing, I think, but the schema can be confusing because the subelements are optional/required.

Reply:
Same for revocation_details